packages/@react-types/dialog/src/index.d.ts

```diff
@@ -36,10 +36,20 @@ export interface SpectrumBaseDialogProps extends HTMLAttributes<HTMLElement> {
 }
 
 export interface SpectrumDialogProps extends DOMProps, StyleProps {
+  /** The contents as semantic elements to display in the dialog. */
```

Comment:
IMO, feel like you can just say "The contents of the dialog." The "semantic elements" bit might make it confusing to the end user